dev-lua/luv/luv-1.30.1.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils unpacker
+
+# e.g. PV = a.b.c.d
+# a.b.c
+MY_BASE="${PV%.*}"
+# d
+MY_PATCH="${PV##*.}"
+# a.b.c-d
+MY_PV="${MY_BASE}-${MY_PATCH}"
```

Comment:
This whole thing can be done in 1 line instead of 7. You've got the right idea, but it doesn't need to be this complicated.
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
So out of my head something like `MY_PV="${PV//-/.}"` should always work. Then there's `ver_rs` with EAPI-7.

Reply:
I always forget about the EAPI-7 `ver_*` functions, fixed and it's much cleaner now :)